src/graphql/queries/colony.graphql

```diff
@@ -1,3 +1,16 @@
+query CombinedColony($name: String!, $address: ID!) {
```

Comment:
I don't think this is the best approach. As far as I understand, there isn't a strong use case for it right now - we tend to know whether we want to get a colony by name or address, and all this is doing is making a redundant duplicated query. 

Why not add a separate, `GetFullyColonyByAddress` query and call the generated hook in `ActionDetailsPage`?

Reply:
Sure